web/libs/editor/src/lib/AudioUltra/Visual/Renderer/Plugins/GridRendererPlugin.ts

```diff
@@ -154,8 +154,8 @@ export class GridRendererPlugin implements RendererPlugin<GridRendererPluginConf
       if (gridFreqs[gridFreqs.length - 1] !== nyquist) gridFreqs.push(nyquist);
     } else if (scale === "mel") {
       const mel = (f: number) => 2595 * Math.log10(1 + f / 700);
-      const invMel = (m: number) => 700 * (Math.pow(10, m / 2595) - 1);
-      const melMax = mel(nyquist);
+      const _invMel = (m: number) => 700 * (10 ** (m / 2595) - 1);
+      const _melMax = mel(nyquist);
```

Comment:
@cloudmark these changes con't appear used anywhere, hence why the linter put a `_` before them. Can we double check and clean up this aspect? Also, note that removing these will likely also make other values unused and so on, I guess I just question whether this code path is doing what is expected.